orchestrator/src/main/java/com/scottlogic/deg/orchestrator/generate/GenerateExecute.java

```diff
@@ -60,14 +64,16 @@
         this.configValidator = configValidator;
         this.profileSchemaValidator = profileSchemaValidator;
         this.profileValidator = profileValidator;
+        this.schemaVersionValidator = schemaVersionValidator;
         this.monitor = monitor;
     }
 
     public void execute() throws IOException {
         configValidator.preProfileChecks(configSource);
-        profileSchemaValidator.validateProfile(configSource.getProfileFile());
 
         Profile profile = profileReader.read(configSource.getProfileFile().toPath());
```

Comment:
There's a few issues here.

to get the schema, it needs to read the profile.
but before the profile is fully read, the schema should have been validated, otherwise we'll get read errors when we should have got validate errors.

the profile validation should be part of the ```profileReader.read()``` call. rather than separate things.
now this is a slight deviation from how it was previously done.

but it has two benefits. 1, the orchestrator doesn't need to know how to do profile specific validation, just the profile does. 2, (probably more important) the 3 different methods calls don't need to be repeated along the 3 different 'execute' entry points to the system.

also this would mean that the 'profile' class would no longer need a schema version.